js/modules/k6/http/http.go

```diff
@@ -60,6 +60,13 @@ func (r *RootModule) NewModuleInstance(vu modules.VU) modules.Instance {
 		}
 	}
 
+	
+	validateArgCount := func(methodName string,args ...sobek.Value){
+	if len(args)>2 {
+		vu.State().Logger.Warningf("%s method has more than two arguments",methodName)
+		}
+	}
```

Comment:
```suggestion
	validateArgCount := func(methodName string,args ...sobek.Value){
	if len(args)>2 {
		vu.State().Logger.Warnf("http.%s method has more than two arguments",methodName)
		}
	}
```

Small suggestion

Reply:
I do agree with @joanlopez that it's better to add `http` prefix. 

Also, how about changing the message to something more actionable, like `http.%s requires two arguments, but %d were provided. Please adjust the input.`